jwst/pipeline/calwebb_spec2.py

```diff
@@ -243,6 +245,18 @@ def process_exposure_product(
                         raise RuntimeError('Cannot determine WCS.')
 
         # Steps whose order is the same for all types of input:
+        # self-calibrate bad/warm pixels, and apply to both background and science
```

Comment:
Minor nit, but to the uninitiated the statement that you're "self calibrating bad/warm pixels" could be misconstrued as indicating you're actually doing something to those pixels to actually calibrate them. When in reality you're just finding and flagging them all. The "self calibration" statement just means you're using the data itself to do some kind of operation on itself. So maybe consider some rewording of this comment.  Oh, and I also really like to see the use of punctuation in multi-line comments, i.e. if lines 248 and 249 are two separate comments, as opposed to one multi-line, run-on sentence, then they need periods at the ends of the lines.  ;-)